### PR TITLE
[ONNX]clamp

### PR DESCRIPTION
**Pull request by xiaxin1998**

### CODE REVIEW

**Comment by xiaxin1998** [comment]
problem: layer not found, I use numpy

File: pyzoo/zoo/pipeline/api/onnx/mapper/clip.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+class ClipMapper(OperatorMapper):
+    def __init__(self, node, _params, _all_tensors):
+        super(ClipMapper, self).__init__(node, _params, _all_tensors)
+
+    def _to_tensor(self):
+        assert len(self.model_inputs) == 1, "Clip accept single input only"
+        min = int(self.onnx_attr['min'])
+        max = int(self.onnx_attr['max'])
+        assert min <= max, "Min must be smaller or equal than Max"
+        clip = Clamp(min, max)
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/clip.py]: you should use `KerasLayerWrapper(Clamp(xxx))` here.

File: pyzoo/zoo/pipeline/api/keras/layers/wrappers.py [hunk 2]
```diff
@@ -99,6 +99,7 @@ class KerasLayerWrapper(ZooKerasLayer):
     creating: createLinear
     >>> kerasLayer = KerasLayerWrapper(linear, input_shape=(100, ))
     creating: createZooKerasKerasLayerWrapper
+    >>> kerasLayer = KerasLayerWrapper(clamp, input_shape=(100, ))
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/keras/layers/wrappers.py]: remove this line?

**Comment by xiaxin1998** [comment]
18888/view/ZOO-PR/job/ZOO-PR-Validation/1403/  Jenkins Passed @zhichao-li

**Pull request merged**